chain/chain/src/store.rs

```diff
@@ -2077,6 +2078,24 @@ impl<'a> ChainStoreUpdate<'a> {
         self.trie_changes.push(trie_changes);
     }
 
+    pub fn save_state_transition_data(
+        &mut self,
+        block_hash: CryptoHash,
+        shard_id: ShardId,
+        partial_storage: Option<PartialStorage>,
+        exact_receipts_hash: CryptoHash,
```

Comment:
what is `exact_receipts_hash` exactly?

Reply:
Good question, added comment to ApplyChunkResult and renamed to applied_receipts_hash.